cookbooks/bcpc/recipes/mysql.rb

```diff
@@ -196,7 +196,7 @@
 max_connections =
   [
     (mysql_nodes.length * 50 + all_nodes.length * 5),
-    200
+    300
```

Comment:
During my initial testing long ago I was getting some error for `MySQL` connections. I increased it by 100 at that time and it worked. The calculation takes max of `[(mysql_nodes.length * 50 + all_nodes.length * 5), 300]`. There was no specific reason to pick `300`.